CODEOWNERS

```diff
@@ -25,3 +25,4 @@ glibc.yaml           @wolfi-dev/foundations-squad
 llvm*.yaml           @wolfi-dev/foundations-squad
 openssh.yaml         @wolfi-dev/foundations-squad
 openssl.yaml         @wolfi-dev/foundations-squad
+nats*.yaml           @wolfi-dev/sustaining-team
```

Comment:
Adding a comment here would be helpful for future travellers.

Reply:
Done. Thank you